style.css

```diff
@@ -0,0 +1,7 @@
+main {
```

Comment:
Please use more CSS styling to make the page more appealing to the viewer. :sparkles: